generators/app/prompts.js

```diff
@@ -25,13 +44,41 @@ module.exports = {
       choices: [
         {
           name: 'factory_girl_rails',
-          checked: true,
+          checked: false,
           value: "gem 'factory_girl_rails'"
         }
       ]
     }]).then(function(answers) {
       this.testGems = answers.gems;
       this.log('\n');
     }.bind(this));
+  },
+
+  env: function() {
+    var defaultEnvFile = ejs.render(this.fs.read(this.templatePath('shared/.env')), {
```

Comment:
In that case would be useful, but maybe its simple enough for a script, without being needed to run a generator when is not creating a new project.